packages/gatsby-remark-code-repls/src/__tests__/gatsby-node.js

```diff
@@ -53,80 +58,95 @@ describe(`gatsby-remark-code-repls`, () => {
       )
     })
 
-    it(`should ignore non JavaScript files in the examples directory`, () => {
-      recursiveReaddir.mockReturnValue([`javascript.js`, `not-javascript.html`])
+    it(`should ignore non JavaScript files in the examples directory`, async () => {
+      readdir.mockResolvedValue([`javascript.js`, `not-javascript.html`])
 
-      createPages(createPagesParams)
+      await createPages(createPagesParams)
 
       expect(fs.readFileSync).toHaveBeenCalledTimes(1)
       expect(fs.readFileSync).toHaveBeenCalledWith(`javascript.js`, `utf8`)
     })
 
-    it(`should error if provided an invalid examples directory`, () => {
+    it(`should error if provided an invalid examples directory`, async () => {
       fs.existsSync.mockReturnValue(false)
 
-      expect(() => createPages(createPagesParams)).toThrow(
-        `Invalid REPL directory specified: "REPL/"`
-      )
+      try {
+        await createPages(createPagesParams)
```

Comment:
```suggestion
        expect(() => createPages(createPagesParams)).toThrow()
```
https://jestjs.io/docs/en/expect#tothrowerror

Reply:
yours is great! 👍